components/CompositionCollectionView/src/ExpressionsFork/Expressions/ExpressionValues/ExpressionValues.Constant.cs

```diff
@@ -0,0 +1,216 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Numerics;
+using Windows.UI;
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations.ExpressionsFork
```

Comment:
Do these things already exist in the toolkit? Can we not just reference them from the existing package?

Reply:
This experiment needs the changes from my branch at https://github.com/CommunityToolkit/WindowsCommunityToolkit/pull/4559, which we never merged to master, that's why I'm including my own fork that is evolving in parallel. I agree that adding a copy of those classes is not ideal, but there doesn't seem to be any good way to add functionality to those core toolkit classes from an experiment (I can't add the Evaluate() method as an extension method because it relies on private properties of the ExpressionNodes).